### PR TITLE
Updated comment to add .php at the end accessTokenEndpoint.

### PR DESCRIPTION
**Pull request by kbagchiGWC**

### CODE REVIEW

File: SwiftVoiceQuickstart/ViewController.swift [hunk 1]
```diff
@@ -11,6 +11,10 @@ import PushKit
 import TwilioVoice
 
 let baseURLString = <#URL TO YOUR ACCESS TOKEN SERVER#>
+/**
+ If your token server is written in PHP, accessTokenEndpoint needs .php extension at the end.
```
Comment (bchen-twilio) [inline SwiftVoiceQuickstart/ViewController.swift]: Format nitpick:
```
/**
 * block comment
 */
```

Comment (kbagchiGWC) [reply SwiftVoiceQuickstart/ViewController.swift]: Was referring to http://nshipster.com/swift-documentation/  :) .. will fix.

Comment (kbagchiGWC) [reply SwiftVoiceQuickstart/ViewController.swift]: And https://developer.apple.com/library/content/documentation/Xcode/Reference/xcode_markup_formatting_ref/ComentBlock.html

File: SwiftVoiceQuickstart/ViewController.swift [hunk 2]
```diff
@@ -11,6 +11,8 @@ import PushKit
 import TwilioVoice
 
 let baseURLString = <#URL TO YOUR ACCESS TOKEN SERVER#>
+// If your token server is written in PHP, accessTokenEndpoint needs .php extension at the end.
+// For example : /accessToken.php
```
Comment (bchen-twilio) [inline SwiftVoiceQuickstart/ViewController.swift]: Should be okay to have just one line.

**Pull request merged**